Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanUtility.cpp

```diff
@@ -364,7 +375,9 @@ namespace webrtc
             {
                 barrier.srcAccessMask = VK_ACCESS_HOST_WRITE_BIT | VK_ACCESS_TRANSFER_WRITE_BIT;
             }
-            barrier.dstAccessMask = VK_ACCESS_SHADER_READ_BIT;
+
+            // Vulkan API validation error?
+            // barrier.dstAccessMask = VK_ACCESS_SHADER_READ_BIT;
```

Comment:
I noticed these validation errors as well, but I believe the mask is correct.
Really strange!

Reply:
Agreed! At this point it's been a long time I was fine-tuning the details so my memory is a bit hazy, but I was trying to find common ground that'll work with the way Unity graphics device handles the layout/state flushes (across multiple versions) and something that was also working with RTX/GTX drivers

Reply:
@BrianHarrisonUnity @karasusan I refreshed my memory on this part a bit and tested against recent enough Unity LTS (2021.3.24ff1) in case something in the Vulkan implementation has changed since the original implementation time. It looks like whatever made me disable the dstAccessMask at the time, it's now safe to revert back.

It's unfortunate that Unity plugin interface doesn't fill `UnityVulkanImage.layout` for us when calling `AccessTexture`, so you'll have to make some assumptions. But this should only be relevant when uploading a texture from the plugin towards Unity. I've kept this this extra fiddling out of the PR for simplicity sake, it's more relevant to video rendering performance with staging texture support.

When running Unity player build on Linux with -force-vulkan-layers, I'm only getting two validation errors during initialization but they're not related to WebRTC.
```
VUID-vkCmdClearAttachments-rect-02682(ERROR / SPEC): msgNum: -1380104337 - Validation Error: [ VUID-vkCmdClearAttachments-rect-02682 ] Object 0: handle = 0x7f97d00b98f0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0xadbd476f | CmdClearAttachments(): pRects[0].rect.extent.width is zero. The Vulkan spec states: The rect member of each element of pRects must have an extent.width greater than 0 (https://vulkan.lunarg.com/doc/view/1.3.250.1/linux/1.3-extensions/vkspec.html#VUID-vkCmdClearAttachments-rect-02682)
VUID-vkCmdClearAttachments-rect-02683(ERROR / SPEC): msgNum: -566636251 - Validation Error: [ VUID-vkCmdClearAttachments-rect-02683 ] Object 0: handle = 0x7f97d00b98f0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; | MessageID = 0xde39d125 | CmdClearAttachments(): pRects[0].rect.extent.height is zero. The Vulkan spec states: The rect member of each element of pRects must have an extent.height greater than 0 (https://vulkan.lunarg.com/doc/view/1.3.250.1/linux/1.3-extensions/vkspec.html#VUID-vkCmdClearAttachments-rect-02683)
```